third_party/terraform/website/docs/r/bigtable_instance.html.markdown

```diff
@@ -31,39 +31,16 @@ resource "google_bigtable_instance" "production-instance" {
 
   cluster {
     cluster_id   = "tf-instance-cluster"
-    zone         = "us-central1-b"
     num_nodes    = 1
     storage_type = "HDD"
   }
 
-  lifecycle {
-    prevent_destroy = true
-  }
-
   labels = {
     my-label = "prod-label"
   }
 }
 ```
 
-## Example Usage - Development Instance
```

Comment:
It looks like the distinction between DEVELOPMENT & PRODUCTION is going away, so +1 this doesn't seem to add any value.